source/software/containers/index.rst

```diff
@@ -188,21 +235,21 @@ Using a container to compile a model
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To build a model within a container, the container must have the compiler and
-all required libraries and library headers.  An easy way to accomplish this, is
-to launch an interactive shell in the container, and build the model as is
+all required libraries and library headers.  An easy way to accomplish this is
+to launch an interactive shell in the container, then build the model as is
 typically done on any system.
 
 .. code-block:: shell
 
-    $ singularity exec container_image.simg bash
+    $ singularity shell container.sif
 
 Using a container to run a parallel job
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Here is an example Slurm run script to run the wrf model with 512 MPI tasks. In
-this example, the :file:`wrf.exe`` file is not included in the
-:file:`hydro.simg` image file.  The :file:`wrf.exe` and :file:`hydro.simg`
-files are all in the working directory of the run.
+Here is an example Slurm script to run the wrf model with 512 MPI tasks. In
```

Comment:
Has this example actually been shown to work on any of the systems supporting Singularity?  It would be reasonable to provide some hints to the user.